Audiobook Player/Player/MoreOptions.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MoreOptions.swift
+//  Audiobook Player
+//
+//  Created by Florian Pichler on 01.04.18.
+//  Copyright © 2018 Florian Pichler.
+//
+
+import UIKit
+
+class MoreOptions {
+    static let shared = MoreOptions()
```

Comment:
I don't think `MoreOptions` should be a singleton, there's no state or variable that it keeps. We could make the function `actionSheet` a class function so it would be called like so: `MoreOptions.actionSheet(actions:...`

Reply:
😅 yes, I went a little overboard there. I think I should be able to put this action sheets back into the PlayerViewController